app/presenters/check/supplementary-data.presenter.js

```diff
@@ -21,6 +24,18 @@ function go (data) {
   }
 }
 
+function _licenceExistsInBilling (billingInvoiceLicences) {
+  if (billingInvoiceLicences) {
+    for (let i = 0; i < billingInvoiceLicences.length; i++) {
+      if (billingInvoiceLicences[i].billingInvoice) {
+        return true
+      }
+    }
```

Comment:
I did a quick test locally (in my console) and I think you can slim this down to the following, whilst IMHO, it still makes sense what it's doing

```suggestion
    return billingInvoiceLicences.some((billingInvoiceLicence) => {
        return billingInvoiceLicence.billingInvoice
    })
```

This might even be the exception to our convention of 'no inline methods'.

```javascript
    return billingInvoiceLicences.some((billingInvoiceLicence) => billingInvoiceLicence.billingInvoice)
```

What do you think?

Reply:
Yep that works, I'll make the change